cmd/collector/main.go

```diff
@@ -70,11 +74,21 @@ func main() {
 	server := thrift.NewServer(ch)
 	server.Register(jc.NewTChanCollectorServer(jaegerBatchesHandler))
 	server.Register(zc.NewTChanZipkinCollectorServer(zipkinSpansHandler))
+
 	portStr := ":" + strconv.Itoa(*builder.CollectorPort)
 	listener, err := net.Listen("tcp", portStr)
 	if err != nil {
 		logger.Fatal("Unable to start listening on channel", zap.Error(err))
 	}
 	ch.Serve(listener)
-	select {}
+
+	r := mux.NewRouter()
```

Comment:
We already use mux elsewhere, I'd like to stay consistent. and yarpc doesn't have nearly the same market penetration to justify a change.